docs/diagrams/mvp.puml

```diff
@@ -0,0 +1,160 @@
+
+box Consumer #ConsumerColor
+    actor Partner1 as "Partner" #white
+    database FileShare1 as "Fileshare"
+    database AssetCatalogConsumer as "Asset\nCatalog" #white
+    participant ConnectorConsumer as "Eclipse\nDataspace\nConnector" #white
+end box
+
+box "Data Eco System (e.g. IDS)"
+    participant Broker as "Metadata Catalog\n(IDS Broker)"
+    participant DAPS as "Identity Provider\n(IDS DAPS & ParIS)"
+end box
+
+box Provider #ProviderColor
+    participant ConnectorProvider as "Eclipse\nDataspace\nConnector" #white
+    database AssetCatalogProvider as "Asset\nCatalog" #white
+    database FileShare2 as "Fileshare"
+    actor Partner2 as "Partner" #white
+end box
+
+hnote over Partner1, Partner2 #Business
+**Precondition**
+Identity registration and verification done and
+connector is configured accordingly (including certificates)
+end note
+
+hnote over Partner1, Partner2 #Business
+**Asset & AssetData**
+Asset - Metadata of AssetData
+AssetData - Unit of Transfer (e.g. one photo or one photo-album)
+end note
+
+== Contract Offering ==
+
+Partner2 -> FileShare2 : Insert AssetData
+Partner2 -> ConnectorProvider : Configure Contract Offer Framework
+rnote right Partner2 #white
+The Contract Offer Frameworks
+outlines how a contract offer
+should look like for a range
+of Assets.
+end note
+Partner2 -> AssetCatalogProvider : Create Asset
+
+
+ConnectorProvider -> Broker : (optional) Register Contract Offer(s)
+Partner1 -> ConnectorConsumer : Query Contract Offers
+alt ways to get provider offers
+        ConnectorConsumer -> Broker : (optional) Query Message
+else
+        ConnectorConsumer -> Broker : (optional) Metadata Request Message
+else
+    ConnectorConsumer -> ConnectorProvider : Metadata Request Message
+else
+    ConnectorConsumer -> ConnectorProvider : Query Message
+end alt
+
+== Contract Negotiation ==
+Partner1 -> ConnectorConsumer : Accept Contract for Data Asset
+ConnectorConsumer <--> ConnectorProvider : Negotiate Contract
+hnote over ConnectorConsumer, ConnectorProvider #Business
+**Limitation of Contract Negotiation**
+Within the MVP contract offers will be accepted and declined automatically.
+Already offered contracts will be accepted, everything else is declined.
+Therefore, the creation of counter offers is not possible.
+
+//The negotiation sequence is shown in another diagram.//
+end note
+hnote over ConnectorConsumer, ConnectorProvider #Business
+**Contract Offer Frameworks**
+These frameworks allow to limit the availability of contract offers to
+specific groups of consumers. As contract offers are accepted automatically, this
+should be the tool of choice to control data access within a dataspace.
+end note
+
+== Data Transfer ==
+Partner1 -> ConnectorConsumer : Initiate Data Transfer of Asset
+alt push
+    ConnectorConsumer -> ConnectorProvider ++: Data Request Message
+    ConnectorProvider -> FileShare2 ++ : Download
+    return AssetData
+    ?<- ConnectorProvider --: Push to fileshare/database/...
```

Comment:
is it intended that this message ends "undefined"? should it end at FileShare1?

Reply:
You are correct, this might be confusing. We had there the problem, that the whole file transfer is in general not so easy to describe. It might be synchron or asynchron, it could be a file transfer, streaming, a rest endpoint, etc.

And that's the reason the end of this message is undefined. The intent was just so how, that the data goes "somewhere".